iree/compiler/Conversion/LinalgToLLVM/LLVMCodeGenOptions.cpp

```diff
@@ -24,23 +24,29 @@ static llvm::cl::opt<bool> clEnableLLVMLinalgOnTensors(
     llvm::cl::desc("Enable the linalg on tensors experimental LLVM path"),
     llvm::cl::init(false));
 
-static llvm::cl::opt<bool> convImg2ColConversion(
+static llvm::cl::opt<bool> clConvImg2ColConversion(
     "iree-codegen-linalg-to-llvm-conv-img2col-conversion",
     llvm::cl::desc("Enable rewriting linalg.conv_2d_input_nhwc_filter_hwcf "
                    "linalg.generic that does img2col buffer packing + "
                    "linag.matmul"),
     llvm::cl::init(false));
 
-static llvm::cl::opt<bool> unfusedFMA(
+static llvm::cl::opt<bool> clUnfusedFMA(
     "iree-codegen-linalg-to-llvm-use-unfused-fma",
     llvm::cl::desc("Enable rewriting llvm.fma to its unfused version."),
     llvm::cl::init(false));
 
+static llvm::cl::opt<bool> clEnableLinalgOnTensorsToVectors(
```

Comment:
These seem unrelated. Can you split them into a different PR?

Reply:
This PR will fail because now vectorization on tensors pass will kick in for statistically shaped tensors and blow up for big tiles.

Reply:
Ah, now I understand. Ok thanks! In reality, should just remove that pass. It is actually not used right now.

Reply:
Correct! this pass with no-op before this PR. We can keep it for now and revisit once explored tensor -> vector pass. I think for matmul if we did the tiling in a separate pass before this pass this one will stay as is.

Reply:
Sure. Approved this PR already.